src/elfpy/simulators.py

```diff
@@ -4,20 +4,13 @@
 
 TODO: rewrite all functions to have typed inputs
 """
-
-
 from importlib import import_module
-
-
-import numpy as np
-
-
+from elfpy.utils.fmt import *  # float→str formatter, also imports numpy as np
```

Comment:
imports should be explicit per file -- so import it here even if it is imported in fmt.